install.sh

```diff
@@ -1,27 +1,102 @@
 #!/bin/bash
+echo -n "Enter your github email: "
+read email
+
+if [ -f "$HOME/.ssh/id_rsa.pub" ]; then
+   sudo cp $HOME/.ssh/id_rsa.pub $HOME/sshkey.txt
```

Comment:
Don't copy the ssh key to the home directory. Just the print the default location ```$HOME/.ssh/id_rsa.pub```